main.go

```diff
@@ -76,6 +76,8 @@ func main() {
 		fmt.Printf("Failed to read configuration: %v\n", err)
 		os.Exit(1)
 	}
+	// Determine the preservation system
+	useArchivematica := cfg.PreservationSystem == temporal.AmWorkerTaskQueue
```

Comment:
I think we should get rid of `useArchivematica` altogether, and pass `cfg.PreservationSystem` directly as a Workflow argument.